app/controllers/ticity_searches_controller.rb

```diff
@@ -1,2 +1,39 @@
 class TicitySearchesController < ApplicationController
+  
+  def index
+    ticity_searches = TicitySearch.all
```

Comment:
this action returns all searches for all users. Is scoping this down to just the logged-in user a planned feature?